src/coreclr/src/vm/callingconvention.h

```diff
@@ -1832,6 +1836,35 @@ class ArgIterator : public ArgIteratorTemplate<ArgIteratorBase>
         return FALSE;
 #endif
     }
+
+    BOOL HasValueTypeReturn()
+    {
+        WRAPPER_NO_CONTRACT;
+
+        TypeHandle thValueType;
+        CorElementType type = m_pSig->GetReturnTypeNormalized(&thValueType);
+        return type == ELEMENT_TYPE_VALUETYPE && !thValueType.IsEnum();
+    }
+
+#if defined(TARGET_X86) && !defined(FEATURE_STUBS_AS_IL)
+    BOOL HasMultiRegReturnStruct()
```

Comment:
Please add a link to the X86 calling convention specification  that indicates this or at a minimum comment this fact on the function.